phoenix-core/src/main/java/org/apache/phoenix/jdbc/PhoenixStatement.java

```diff
@@ -1855,25 +1893,39 @@ public MutationPlan compileMutation(String sql) throws SQLException {
         return compileMutation(stmt, sql);
     }
 
-    public QueryLogger createQueryLogger(CompilableStatement stmt, String sql) throws SQLException {
-        if (connection.getLogLevel() == LogLevel.OFF) {
-            return QueryLogger.NO_OP_INSTANCE;
-        }
-
-        boolean isSystemTable = false;
-        if(stmt instanceof ExecutableSelectStatement) {
+    public boolean checkIgnoreQueryAudit(CompilableStatement stmt) {
```

Comment:
This is now applies to al logs, so we shouldn't call it checkIgnoreQuery**Audit**